### PR TITLE
Layout switching configuration with dynamic property support

### PR DESCRIPTION
**Pull request by gayanSandamal**
## Description

This feature includes,

1. Switching between 5 different layouts `row-left`,  `row-left-swapped`, `row-spread`,  `row-spread-swapped` which is similar with `ui-text` node
2. Dynamic property support for the newly added layout property
3. Docs updated for the newly added property `Layout` and marked `dynamic: true` in the docs for the existing dynamic properties which haven't marked previously
4. Necessary E2E tests are added

## Related Issue(s)

https://github.com/FlowFuse/node-red-dashboard/issues/1236

## Checklist

 - [x] I have read the [contribution guidelines](https://github.com/FlowFuse/flowfuse/blob/main/CONTRIBUTING.md)
 - [x] Suitable unit/system level tests have been added and they pass 
 - [x] Documentation has been updated
    - [ ] Upgrade instructions
    - [ ] Configuration details
    - [ ] Concepts
 - [ ] Changes `flowforge.yml`?
    - [ ] Issue/PR raised on `FlowFuse/helm` to update ConfigMap Template
    - [ ] Issue/PR raised on `FlowFuse/CloudProject` to update values for Staging/Production

## Labels

 - [ ] Includes a DB migration? -> add the `area:migration` label

### CODE REVIEW

**Comment by gayanSandamal** [comment]
> Ready for review @gayanSandamal?

Yes, I was waiting for tests to pass. I'm marking this as ready for review

**Comment by gayanSandamal** [comment]
The PR is updated. only the usual `❌  widgets/control.spec.js` fails

File: nodes/widgets/ui_switch.html [hunk 1]
```diff
@@ -194,6 +206,37 @@
...
+                    <span class="nr-db-switch-layout-label">label</span>
+                    <span class="nr-db-switch-layout-value">switch</span>
+                    <div class="nr-db-switch-layout-checkbox"></div>
+                </a>
+                <a href="#" class="nr-db-switch-layout nr-db-switch-layout-row nr-db-switch-layout-row-spread-swapped">
+                    <span class="nr-db-switch-layout-value">switch</span>
+                    <span class="nr-db-switch-layout-label">label</span>
+                    <div class="nr-db-switch-layout-checkbox"></div>
+                </a>
+            </div>
```
Comment (joepavitt) [inline nodes/widgets/ui_switch.html:237]: I don't understand _why_, but for some reason, even though this code is correct, it renders in the Node-RED Editor with the last option reversed in it's rendering:

Screenshot 2024-09-24 at 16 46 32

So there are two options that _look_ the same. The HTML rendered to the DOM is as rendered, but not as defined in the underlying HTMl file. It's incredibly strange.

Functionally, the PR is great. Docs are good, but this is a strange one

Comment (joepavitt) [reply nodes/widgets/ui_switch.html:237]: I can see the problem, CSS issues. Fixing  myself now, and will use `display: flex` as it's cleaner

Comment (gayanSandamal) [reply nodes/widgets/ui_switch.html]: > I can see the problem, CSS issues. Fixing myself now, and will use `display: flex` as it's cleaner

Thanks fixing this Joe

**Review by joepavitt — APPROVED** [review]

**Pull request merged**